### ISSUE TITLE
Missing table of content title

### ISSUE THREAD

**Issue by 0xswitch** [retypeapp/retype#440]
hi,

H1 and > H4 are missing from table of content (right sidebar), is it intended ? 

Thanks.

**Comment by geoffreymcgill** [comment]
yes, correct. Only H2` and `H3` headings will be used to generate the right sidebar table of contents.

Only one H1 should exist on the page and that should be the page title. 

We decided H4 was too granular and would create unnecessary noise/clutter within the table of contents.

**Comment by 0xswitch** [comment]
thanks for the feedback. Agree for H1, however it's possible to let's user decides until which title the content table would go ? It would be very useful when messing wery quite long and detailed pages. Thanks!

**Comment by noraj** [comment]
Agree, for example for the ToC on hexo (https://hexo.io/docs/helpers#toc) there is a `min_depth` and `max_depth` to control the heading level displayed.

**Comment by hiddentao** [comment]
Just hit this issue myself with H4 tags. Would be good to be able to at least expand the depth to H4 and beyond if necessary via a page config parameter.

**Comment by geoffreymcgill** [comment]
Related to #598 feature request.
